src/RCAT_main.py

```diff
@@ -526,20 +527,21 @@ def get_mod_data(model, mconf, tres, var, cfactor, prep_func):
     return outdata
 
 
-def get_obs_data(obs, var, cfactor, sy, ey, mns):
+def get_obs_data(metadata_file, obs, var, cfactor, sy, ey, mns):
     """
     Open obs data files.
     """
-    import observations_metadata as obsmeta
-    obs_dict = obsmeta.obs_data()
+    from importlib.machinery import SourceFileLoader
```

Comment:
Do we need to install a new module for this?

Reply:
No, importlib is part of Python's standard library of modules.